apps/meteor/packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -7,8 +7,8 @@
   "__usersCount__people_will_be_invited": "__usersCount__ people will be invited",
   "__username__is_no_longer__role__defined_by__user_by_": "__username__ is no longer __role__ by __user_by__",
   "__username__was_set__role__by__user_by_": "__username__ was set __role__ by __user_by__",
-  "This_room_encryption_has_been_enabled_by__username_": "This room's encryption has been enabled by __username__",
-  "This_room_encryption_has_been_disabled_by__username_": "This room's encryption has been disabled by __username__",
+  "This_room_encryption_has_been_enabled_by__username_": "enabled E2E Encryption for this room",
+  "This_room_encryption_has_been_disabled_by__username_": "disabled E2E Encryption for this room",
```

Comment:
if `__username__` is going to be removed from these translations the code needs to be updated as well.

ideally I'd say to create a new string instead of changing these.

Reply:
And this is a breaking change on mobile apps. We need to keep supporting old servers.

Reply:
> if `__username__` is going to be removed from these translations the code needs to be updated as well.
> 
> ideally I'd say to create a new string instead of changing these.

Agree this is a better option. We were first thinking of keeping the code the same + changing the strings so as not to "lose" the translations we already have, but I see this way it'd be difficult to track the languages with outdated translations :/
So we'll create new strings and change the code to work only with the data from our new system messages. It's still going to be a breaking change on mobile apps anyways, right @diegolmello ?

Reply:
> So we'll create new strings and change the code to work only with the data from our new system messages. It's still going to be a breaking change on mobile apps anyways, right @diegolmello ?

It won't if we sync the implementation.

Reply:
updated!